drivers/serial/uart_ns16550.c

```diff
@@ -1374,7 +1918,7 @@ static const struct uart_driver_api uart_ns16550_driver_api = {
 		IF_ENABLED(DT_INST_NODE_HAS_PROP(n, pinctrl_0),                      \
 			(.pincfg = PINCTRL_DT_DEV_CONFIG_GET(DT_DRV_INST(n)),))      \
 		IF_ENABLED(DT_INST_NODE_HAS_PROP(n, resets),                         \
-			(.reset_spec = RESET_DT_SPEC_INST_GET(n),))
+			(.reset_spec = RESET_DT_SPEC_INST_GET(n),))		     \
```

Comment:
Please remove this change if not adding a new line after.

Reply:
Removed